tests.py

```diff
@@ -14,6 +16,7 @@
 
 class SnapPassTestCase(TestCase):
 
+    @patch('redis.client.StrictRedis', mock_strict_redis_client)
```

Comment:
I think I understand. The patching is used here because there's no another way for this test case to always use `mock_strict_redis_client` should some kind of dependency injection, etc. when `MOCK_REDIS` isn't defined. Is that right?

Reply:
Correct, because of how the client is initialized in the main logic, this is the only way I found to replace all instances of the `redis_client` with the mock one.